### PR TITLE
[W5.7] [T9-1] Julius Putra Tanu Setiaji

### PR DESCRIPTION
**Pull request by indocomsoft**

### CODE REVIEW

**Comment by indocomsoft** [comment]
@taneliang @JayPeeTeeDee @tuesmiddt @zxjtan  (since I can't assign reviewer)

File: test/java/seedu/addressbook/parser/ParserTest.java [hunk 1]
```diff
@@ -175,14 +176,14 @@ public void parse_findCommandInvalidArgs_errorMessage() {
     }
 
     @Test
-    public void parse_findCommandValidArgs_parsedCorrectly() {
+    public void parse_findCommandValidArgs_parsedCorrectlyAsUppercase() {
```
Comment (taneliang) [inline test/java/seedu/addressbook/parser/ParserTest.java:179]: Any reason you only renamed this test but not `parse_findCommandDuplicateKeys_parsedCorrectly`?

Comment (indocomsoft) [reply test/java/seedu/addressbook/parser/ParserTest.java:179]: Oops, an oversight on my part.

File: test/expected.txt [hunk 2]
```diff
@@ -213,8 +213,9 @@
 || 0 persons listed!
 || ===================================================
 || Enter command: || [Command entered:  find betsy]
+|| 	1. Betsy Choo Tags: [secretive]
 || 
-|| 0 persons listed!
+|| 1 persons listed!
```
Comment (novellius) [inline test/expected.txt:218]: Good that you've updated the I/O tests.

File: test/java/seedu/addressbook/commands/FindCommandTest.java [hunk 3]
```diff
@@ -25,8 +25,8 @@ public void execute() throws IllegalValueException {
         //same word, same case: matched
         assertFindCommandBehavior(new String[]{"Amy"}, Arrays.asList(td.amy));
 
-        //same word, different case: not matched
-        assertFindCommandBehavior(new String[]{"aMy"}, Collections.emptyList());
+        //same word, different case: also matched
+        assertFindCommandBehavior(new String[]{"aMy"}, Arrays.asList(td.amy));
```
Comment (novellius) [inline test/java/seedu/addressbook/commands/FindCommandTest.java:29]: Good that you've updated the JUnit tests too.

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -79,14 +79,17 @@ Format: `find KEYWORD [MORE_KEYWORDS]`
...
 ====
 
 Examples:
 
 * `find John` +
-Returns `John Doe` but not `john`.
+Returns `John Doe`.
+
+* `find john` +
+Returns `John Doe`
```
Comment (novellius) [inline docs/UserGuide.adoc:92]: Great that you've updated the User Guide and the examples.

**Review by novellius — APPROVED** [review]
Overall, good work @indocomsoft . Please read the comments for suggestions. Please close the PR after reading them.

**Pull request closed without merging**